<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paysucc</class>
 <widget class="QDialog" name="paysucc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;
</string>
  </property>
  <widget class="QLineEdit" name="departure_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="destination_lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:rgb(130,130,210);
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;购票成功：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="departurelabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="destinationlabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发车时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/tuxing.png);
border:none;
background-repeat:no-repeat;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="train_number">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>departure_lineEdit</zorder>
  <zorder>destination_lineEdit</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>departurelabel</zorder>
  <zorder>destinationlabel</zorder>
  <zorder>exit</zorder>
  <zorder>label</zorder>
  <zorder>date</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
